fprodents/workflows/base.py

```diff
@@ -306,15 +305,15 @@ def init_single_subject_wf(subject_id):
 
         bold_ref_wf = init_epi_reference_wf(
             auto_bold_nss=True,
-            omp_nthreads=config.nipype.omp_nthreads
+            omp_nthreads=config.nipype.omp_nthreads,
+            n4_iter=4,
+            adaptive_bspline_grid=True,
+            shrink_factor=1,
```

Comment:
Please add some documentation about this decision on the docstring or as a comment above (or both)

Reply:
Added a comment for now